app/models/user.rb

```diff
@@ -470,6 +470,11 @@ def coauthored_works
     return @coauthored_works
   end
 
+  # checks that only collections a user has created are deleted.
```

Comment:
Comment might be superfluous? Method name and definition are already relatively explicit.

Reply:
I'm not actually sure the comment is clear -- we're not really checking the deletion of anything here, are we? We're returning an array of collections that are owned by this user and only this user, like `sole_authored_works` above is described as `# Returns array of works where the user is the sole author`. The actual deletion comes at some other point.